src/cdk-experimental/combobox/combobox-panel.ts

```diff
@@ -24,13 +27,20 @@ export class CdkComboboxPanel<T = unknown> {
   contentId: string = '';
   contentType: AriaHasPopupValue;
 
-  constructor(readonly _templateRef: TemplateRef<unknown>) {}
+  constructor(
+    readonly _templateRef: TemplateRef<unknown>,
+    readonly _elementRef: ElementRef<HTMLElement>
+  ) {}
 
-  /** Tells the parent combobox to closet he panel and sends back the content value. */
+  /** Tells the parent combobox to close the panel and sends back the content value. */
   closePanel(data?: T) {
     this.valueUpdated.next(data);
   }
 
+  focusContent() {
+    document.getElementById(this.contentId)?.focus();
```

Comment:
I would add a `TODO` here to use an injected document and a separate TODO to potentially switch this to use focus trapping from cdk/a11y